analyzers/tests/SonarAnalyzer.UnitTest/Rules/Hotspots/ExecutingSqlQueriesTest.cs

```diff
@@ -33,22 +33,25 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ExecutingSqlQueriesTest
     {
+        private readonly VerifierBuilder builderCs = new VerifierBuilder().AddAnalyzer(() => new CS.ExecutingSqlQueries(AnalyzerConfiguration.AlwaysEnabled));
+        private readonly VerifierBuilder builderVb = new VerifierBuilder().AddAnalyzer(() => new VB.ExecutingSqlQueries(AnalyzerConfiguration.AlwaysEnabled));
+
 #if NETFRAMEWORK // System.Data.OracleClient.dll is not available on .Net Core
 
         [TestMethod]
         public void ExecutingSqlQueries_CS_Net46() =>
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\Hotspots\ExecutingSqlQueries_Net46.cs",
-                new CS.ExecutingSqlQueries(AnalyzerConfiguration.AlwaysEnabled),
-                GetReferencesNet46(Constants.NuGetLatestVersion));
+            builderCs
+                .AddPaths(@"Hotspots\ExecutingSqlQueries_Net46.cs")
```

Comment:
Please use `.WithBasePath` on the fields to avoid repeating `@"Hotspots\`
```suggestion
                .AddPaths("ExecutingSqlQueries_Net46.cs")
```